docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+
+- **You might have less power.** Compared to a fixed-sample analysis, sequential analysis has less power, meaning it's less likely to detect a real effect. However, this loss in power is mostly concentrated in really small sample sizes, and it can be mitigated by using techniques like [CUPED](cuped.md). Furthermore, fixed-sample analysis only has more power if the power analysis used to set the predetermined sample size is actually correct: if the treatment group has a higher than expected variance for a given metric, for example, you'll end up being less powered than you anticipated—and the only thing you can do in that case is rerun the experiment from scratch. With sequential analysis, if you have less power than you'd like you can always choose to keep running the experiment to collect more sample, without affecting the statistical guarantees
+- **You cannot pick and choose your results.** Although you can still make a decision using partial data when a problem invalidates data after a certain point in time, you still, as with fixed-sample analysis, cannot run an experiment on a whole population and then choose to ship it to a subpopulation—at least, not without increasing your false positive rate. For example, if you find that a treatment has a big effect with Canadian users but no effect with users overall or in other countries, you cannot ship that variant just to Canadian users without violating the statistical guarantees on the false positive rate.
+
+
+### Bayesian analysis
+
+Both fixed-sample and sequential methods described above use a [frequentist](https://en.wikipedia.org/wiki/Frequentist_inference) approach, where we test how likely it would be to see the observed data under the assumption that the treatment and the control were identical. In contrast, the [Bayesian](https://en.wikipedia.org/wiki/Bayesian_inference) approach takes the data as given and tries to construct a distribution for the *true* lift, given what we've observed. More formally, the Bayesian method starts with a *prior distribution* for the lift, which describes our beliefs about what the lift might be *before we run the experiment*. Then, we use the data gathered in the experiment to *update* that prior and produce a *posterior distribution* (so called because it comes *after* the data), which describes our *new* beliefs about what the lift might be, given both the prior we started with and the data we've observed.
+
+The prior we use is described specifically in the [nitty-gritty](#statistical-nitty-gritty) section below, but in essence we set our pre-experiment belief to be that the lift on any given metric will be, on average, zero, that for 50% of experiments the lift will fall between -21% and +21%, and that for 95% of experiments the lift will fall between -62% and +62%; if your experiments tend to show bigger lifts in either direction, then our Bayesian confidence intervals[^6] might be too narrow (biasing toward showing an effect when there is none).
```

Comment:
Side note: I think these priors are slightly wide for what we've seen in data, but we can adjust later